### DISCUSSION TITLE
New in pre-reease: improved tab selection / switching with `Ctrl + Tab`.

### DISCUSSION THREAD

**Discussion by kjk** [sumatrapdfreader/sumatrapdf#4482] [Announcements]
To improve working with large number of tabs, `Ctrl + Tab` and `Ctrl + Shift + Tab` now shows overlay window with list of tabs. 

If you just press `Ctrl + Tab` and `Ctrl + Shift + Tab` on release it acts like before i.e. switches to selected tab.

It does have additional capability:
* you can use mouse click to select a tab from list
* you can press `Space` to switch to sticky mode

In sticky mode you can narrow down the list with search, navigate with up / down arrows, select a tab with `Enter` and cancel with `Escape`.

Another way to select tabs this way is with `Ctrl + K` (command palette) and `@`.

You can also create a keyboard shortcut for even more direct use e.g.:

```
Shortcuts [
	[
		Cmd = CmdCommandPalette @
		Key = 8
	]
]
```

This functionality is implemented as [commands](https://www.sumatrapdfreader.org/docs/Commands) `CmdNextTabSmart` and `CmdPrevTabSmart`.

If you prefer old behavior, without the overlay window, you can use `Shortcuts` [advanced setting](https://www.sumatrapdfreader.org/docs/Advanced-options-settings):

```
Shortcuts [
  [
    Cmd = CmdNextTab
    Key = Ctrl + Tab
  ]
  [
    Cmd = CmdPrevTab
    Key = Ctrl + Shift + Tab
  ]
]
```

You really have to try it to experience it. Here's how it looks in actions:

![Screenshot_1](https://github.com/user-attachments/assets/dc8d8fda-f88c-4000-9507-1486f8998c48)

Available in latest [pre-release](https://www.sumatrapdfreader.org/prerelease) build.

**Comment by milen-prg** [comment]
Make scrollable recent opened files in the home tab, additionally it can have chose to be previews like now or detailed list.
Using of Ctrl+K (where are even commands) is not convenient.

**Reply by GitHubRulesOK** [reply]
@milen-prg 
you can set key to whatever but for example # and also an icon like this
![image](https://github.com/user-attachments/assets/28e0416a-c226-41f9-9f16-15434d3e2c94)
Click or press # all the history is there.
![image](https://github.com/user-attachments/assets/22bf85fe-83d3-4111-be7d-7cd8342788ca)
